RELEASE-NOTES.md

```diff
@@ -4,6 +4,12 @@ section: overview
 releaseNoteTOC: true
 ---
 
+## 2020.07 release notes (2020-06-05)
+Packages released:
+- @patternfly/patternfly@v4.10.31
+
+This is our major release. Check out our [upgrade guide](/documentation/core/overview/upgrade-guide) for a list of breaking changes!
```

Comment:
does this link need a `//patternfly.org/v4` prefix?

Reply:
Nope! We have other links that work the same way: 
![image](https://user-images.githubusercontent.com/47335686/83906927-6e5ccb80-a732-11ea-8f7a-8ee5e3573c55.png)

The prefix gets handled automagically in org.